google/cloud/datastore/aggregation.py

```diff
@@ -466,38 +471,67 @@ def _next_page(self):
             "project_id": self._aggregation_query.project,
             "partition_id": partition_id,
             "read_options": read_options,
-            "aggregation_query": query_pb,
+            "aggregation_query": self._build_protobuf(),
```

Comment:
just for my understanding, why is this changing?

Reply:
There was a lot of duplicated code in the [original implementation](https://github.com/googleapis/python-datastore/blob/c86b51e7e56ad52c6ee9ca3c9de2db5553e4ec0a/google/cloud/datastore/aggregation.py#L465)  (see the duplicated request dictionaries)

I did some refactoring of these methods so that building the request and calling `run_aggregation_query` only happens in one place. And I also ended up removing the `query_pb` variable, since I found it a little confusing tracking the state between `query_pb`, `old_query_pb`, and `request["query"]`